pycalphad/core/equilibrium.py

```diff
@@ -79,7 +81,7 @@ def _eqcalculate(dbf, comps, phases, conditions, output, data=None, per_phase=Fa
     """
     if data is None:
         data = equilibrium(dbf, comps, phases, conditions, to_xarray=False)
```

Comment:
We don't call `_eqcalculate` without data internally. Should we support this? If we want to keep this, we need to pass through the arguments like `models` and `parameters`.